seed/data/dev/01200_reductions.js

```diff
@@ -33,26 +35,42 @@ exports.seed = function (knex, Promise) {
       archivedFromDate.setDate(effectiveFromDate - 360)
       archivedToDate.setDate(effectiveToDate - 365)
 
+      // Create deleted reduction record
+      var deletedFromDate = new Date()
+      var deletedToDate = new Date()
+      var deletedStatus = 'DELETED'
+      deletedFromDate.setDate(effectiveFromDate - 365)
+      deletedToDate.setDate(effectiveToDate + 365 * 10)
```

Comment:
Why is this * 10?

Reply:
We want the effective to date to be very far in the future (10 years in this case).  This is because the test gets any records where effectiveto > today.  If we had a closer effective to date the test would work today but not in the near future

Reply:
Ok